components/Trade/Advanced/RightPanel/AccountDetails/index.tsx

```diff
@@ -313,16 +313,14 @@ const Fills: React.FC<{
         <STable headings={['Time', 'Side', 'Price', 'Amount']}>
             <tbody>
                 {filledOrders.map((order, index) => {
-                    const now = Date.now();
-                    const price = order.price;
                     return (
                         <TRow key={`filled-order-${index}`}>
-                            <TData>{timeAgo(now, parseInt(order.timestamp) * 1000)}</TData>
+                            <TData>{timeAgo(Date.now(), parseInt(order.timestamp) * 1000)}</TData>
                             <TData className={!!order.position ? 'ask' : 'bid'}>
                                 {!!order.position ? 'Short' : 'Long'}
                             </TData>
-                            <TData>{toApproxCurrency(price)}</TData>
-                            <TData>{order.amount.toNumber()}</TData>
+                            <TData>{toApproxCurrency(order.price)}</TData>
+                            <TData>{round(order.amount.toNumber())}</TData>
```

Comment:
round rounds it to the closest integer. In js you have a few options. toFixed rounds it to the nearest fixed decimals, ie 1.toFixed(2) is going to be 1.00 but 1.0123131.toFixed(2) will be 1.01. toPrecision sets the number to a specific length. eg 5.6.toPrecision(2) is 5.6 as is 5.6123123.toPrecision(2) see [here](https://www.c-sharpcorner.com/blogs/difference-between-tofixed-and-toprecision-in-javascript1#:~:text=toFixed()%20returns%20digits%20before,count%20starts%20before%20decimal%20point.)

Reply:
```suggestion
                            <TData>{order.amount.toFixed(2)}</TData>
```
In saying that I would do

Reply:
oh you created a round function

Reply:
still I wouldnt use our own round function, kind of pointless that also just rounds it to the nearest 100